example/ck_tile/37_transpose/transpose_policy.hpp

```diff
@@ -132,6 +132,7 @@ struct TransposePolicy
         constexpr index_t kSecondDimStrSub     = kSecondRepetitions / kSecondDimIterations;
 
         constexpr auto xdllevel_dstr_encoding = make_transposed_distr_encode<DataType,
+                                                                             16,
```

Comment:
Could we descriptively name instead of hardcoding a literal constant?

Reply:
Updated